BuildToolsVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.26-prerelease-00524-01
```

Comment:
Yes, if 
1. build-tools starts writing out a build-info file to dotnet/versions.
2. We teach update-dependencies to look for that build-info file, and update this file as necessary.

---
In reply to: [68859625](https://github.com/dotnet/cli/pull/3719#discussion_r68859625) [](ancestors = 68859625)